config-connector/solutions/iam/kpt/member-iam/Kptfile

```diff
@@ -0,0 +1,56 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: .
+packageMetadata:
+  shortDescription: sample description
+openAPI:
+  definitions:
+    io.k8s.cli.setters.project-id:
+      x-k8s-cli:
+        setter:
+          name: project-id
+          value: ${PROJECT_ID?}
+          setBy: PLACEHOLDER
+      description: The target project where you want to create service accounts and
+        grant permissions.
+    io.k8s.cli.substitutions.project-id-in-sa-sub:
+      x-k8s-cli:
+        substitution:
+          name: project-id-in-sa-sub
+          pattern: serviceAccount:SERVICE_ACCOUNT_NAME_SETTER@PROJECT_ID_SETTER.iam.gserviceaccount.com
+          values:
+          - marker: PROJECT_ID_SETTER
+            ref: '#/definitions/io.k8s.cli.setters.project-id'
+          - marker: SERVICE_ACCOUNT_NAME_SETTER
+            ref: '#/definitions/io.k8s.cli.setters.service-account-name'
+    io.k8s.cli.substitutions.project-id-sub:
+      x-k8s-cli:
+        substitution:
+          name: project-id-sub
+          pattern: projects/PROJECT_ID_SETTER
+          values:
+          - marker: PROJECT_ID_SETTER
+            ref: '#/definitions/io.k8s.cli.setters.project-id'
+    io.k8s.cli.setters.service-account-name:
+      x-k8s-cli:
+        setter:
+          name: service-account-name
+          value: member-iam-test
```

Comment:
Should perhaps have a different name, such as member-iam-service-account

Reply:
I believe the resource kind is sufficient for customers to recognize that this is a service account, and we don't need to emphasize it in the name.